charts/aws-ebs-csi-driver/templates/controller.yaml

```diff
@@ -350,6 +350,18 @@ spec:
             - --csi-address=$(ADDRESS)
             - --v={{ .Values.sidecars.resizer.logLevel }}
             - --handle-volume-inuse-error=false
+            - --leader-election={{ (.Values.sidecars.resizer.leaderElection).enabled }}
```

Comment:
This line needs to be modified to:
```
- --leader-election={{ .Values.sidecars.resizer.leaderElection.enabled | required "leader election state for resizer is required, must be set to true || false." }}
```

Reply:
I had to remove it because customers upgrading from a previous version may not have this value set in `values.yaml` which will cause the upgrade to fail. The helm-test workflow is failing due to this.